src/context.ts

```diff
@@ -65,6 +66,8 @@ class TelegrafContext {
   readonly updateSubTypes: ReadonlyArray<typeof MessageSubTypes[number]>
   /** @deprecated */
   private contextState?: any
+  scene: any
+  wizard!: WizardContext<any>
```

Comment:
Good question! I don't think so.

First of all, any usage of `any` can and should be improved because `any` is always just an intermediate migration type.

Second, `wizard` is regularly assumed to be a member that is truthy. Most often, no checks are performed. If it is possible that these values are nullish, this would unveil potential bugs that need fixing. I mainly focused on `typing the library in the ways it works as of today` and not `typing the library in the way it makes sense`. Sometimes that coincides, sometimes it does …

Do you have a suggestion here?

Reply:
They're assigned right here: https://github.com/telegraf/telegraf/blob/5dec6927eab19df2efe02394cfa51cc6ce0eb825/src/scenes/wizard/index.ts#L50 https://github.com/telegraf/telegraf/blob/f78c6eff3065ce5348397d64c139c2fe52754089/src/stage.ts#L37

Reply:
I believe this (from #1113) would be helpful: https://github.com/telegraf/telegraf/blob/3e2d4adb328457d2841a4c3a2ed51b56cead8caf/src/types.ts#L21-L26

Reply:
I'm working on this.

Reply:
I hope they won't turn out contravariant